doc-content/enterprise-only/admin-and-config/managing-business-central-using-DB-usergroupcallback-ref.adoc

```diff
@@ -0,0 +1,50 @@
+[id='managing-business-central-using-DB-usergroupcallback-ref']
+= Database UserGroupCallback implementation
+
+In database `UserGroupCallback` implementation, you must create the required database. You can use this implementation by configuring the respective database properties in one of the following ways:
+
+* Programmatically: build a properties object with the respective `DBUserGroupCallbackImpl` properties and create `DBUserGroupCallbackImpl` with the properties object as its parameter.
```

Comment:
I would suggest capitalizing the first letter of the word after the colon. 

Suggested rewrite:
* Programmatically: Build a properties object with DBUserGroupCallbackImpl properties and create DBUserGroupCallbackImpl using the same properties object as its parameter.